order/buyer/receive/README.md

```diff
@@ -126,25 +126,37 @@ The supplier may check, change and add item details if they are not correct or i
 * `salesOrderNumber`: the sales order number as known in the supplier's ERP system
 * `salesOrderPosition`: the position within the supplier's sales order
 * `description`: a free format additional description of this line by the supplier
-* `proposal`: the supplier can propose a different delivery schedule and prices, see below
+* `requests`: the supplier can request different delivery schedule, prices and charge lines, see below
 * `properties`: are key-value based custom fields, added by the supplier
 * `notes`: are simple custom fields, added by the supplier
 * `documents`: contain meta data and link of attached documents by the supplier.  
 
 {% page-ref page="download-document.md" %}
 
-## Supplier proposal
+### Supplier requests
 
-`proposal`: in stead of accepting or rejecting an order line, the supplier can alternatively propose a different delivery schedule and prices.
+* `requests.proposal`: the supplier has proposed a different delivery schedule, prices and/or charge lines compared to the issued order line.
+* `requests.reopenRequest`: the supplier requests to reopen the confirmed order line. The supplier has requested a different delivery schedule, prices and/or charge lines compared to the confirmed order line.
+* `deliverySchedule`: requested alternative delivery schedule, see below
```

Comment:
Perhaps somehow make clear that this is part of a `proposal` or `reopenRequest`?

Reply:
Refactored paths to indents.